internal/cmd/run/puppeteer.go

```diff
@@ -102,7 +102,7 @@ func runPuppeteerInDocker(p puppeteer.Project, testco testcomposer.Client, rs re
 	log.Info().Msg("Running puppeteer in Docker")
 	printTestEnv("docker")
 
-	cd, err := docker.NewPuppeteer(p, &testco, &testco, &rs, &rs, createReporters(p.Reporters))
+	cd, err := docker.NewPuppeteer(p, region.None, &testco, &testco, &testco, &rs, &rs, createReporters(p.Reporters))
```

Comment:
Why `region.None`?

Reply:
I was using regio region.Region for building URL, but URL now comes in a proper format and I forgot to delete region variable. I fixed this issue.